### PR TITLE
Feature/new event title

### PR DESCRIPTION
**Pull request by gordonzhang17**
- change redux-persist to only include user slice
- add event title to events

### CODE REVIEW

File: parkhangs-webapp/src/components/EventForm.js [hunk 1]
```diff
@@ -18,13 +18,19 @@ class EventForm extends React.Component {
         super(props)
 
         this.state = {
+            eventTitle: this.props.eventTitle || null,
             eventDetail: this.props.eventDetails || null,
             eventStartTime: this.eventStartTime(),
             eventEndTime: this.eventEndTime(),
-            eventDate: this.dayPickerDateFormat(),
+            eventDate: this.eventDate(),
```
Comment (lydia-zheng) [inline parkhangs-webapp/src/components/EventForm.js:25]: I made this change in my branch as well

File: parkhangs-webapp/src/components/ParksCalendar.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
 import React from 'react'
-import { connect } from 'react-redux'
-import { Calendar, momentLocalizer, Views } from 'react-big-calendar'
+import {connect} from 'react-redux'
```
Comment (cmaija) [inline parkhangs-webapp/src/components/ParksCalendar.js:2]: 🙃

File: parkhangs-webapp/src/features/events/eventsSlice.js [hunk 3]
```diff
@@ -117,20 +117,24 @@ const eventSlice = createSlice({
 
             state.updatingEvent = false
             state.error = null
-        },
+        }
+        ,
```
Comment (cmaija) [inline parkhangs-webapp/src/features/events/eventsSlice.js:121]: lol

File: parkhangs-webapp/src/features/events/eventsSlice.js [hunk 4]
```diff
@@ -117,20 +117,24 @@ const eventSlice = createSlice({
...
+
+        updateEventByIdFailure(state, action) {
 
-        updateEventByIdFailure (state, action) {
             state.updatingEvent = false
             state.error = action.payload
-        },
+        }
+        ,
```
Comment (cmaija) [inline parkhangs-webapp/src/features/events/eventsSlice.js:128]: what is happening

File: parkhangs-webapp/src/features/events/eventsSlice.js [hunk 5]
```diff
@@ -117,20 +117,24 @@ const eventSlice = createSlice({
...
 
-        deleteEventStart (state, action) {
+        deleteEventStart(state, action) {
             state.deletingEvent = true
             state.error = null
-        },
+        }
+        ,
```
Comment (cmaija) [inline parkhangs-webapp/src/features/events/eventsSlice.js:134]: please put these commas back and also all of the other weird spacing stuff in this file

File: parkhangs-webapp/src/components/EventForm.js (hunk 1 above)
Comment (gordonzhang17) [reply parkhangs-webapp/src/components/EventForm.js:25]: yeah I added this after I asked you about it

File: parkhangs-webapp/src/components/ParksCalendar.js (hunk 2 above)
Comment (gordonzhang17) [reply parkhangs-webapp/src/components/ParksCalendar.js:2]: will fix

File: parkhangs-webapp/src/components/EventForm.js (hunk 1 above)
Comment (lydia-zheng) [reply parkhangs-webapp/src/components/EventForm.js:25]: Could you just remove this one line? I’m going to merge in and I think aside from this there should be no merge conflicts ... or I’ll deal with it lol 

Everything looks good now

**Pull request merged**